### PR TITLE
Ada1 41

### PR DESCRIPTION
**Pull request by JMB2707**
There appear to be some UI issues with users page. 
unsure if this is caused by this branch or another earlier one

### CODE REVIEW

File: AgentDiscoveries-Frontend/app/src/components/admin/entity.jsx [hunk 1]
```diff
@@ -45,4 +49,15 @@ export default class Entity extends React.Component {
             </Link>
         );
     }
+    getDelButton() {
+        return (
+            //call apidelete
+            <Button type='button' onClick={()=>this.deleteEntity()}>Delete</Button>
+            //TODO: refresh page
```
Comment (RogVan) [inline AgentDiscoveries-Frontend/app/src/components/admin/entity.jsx:56]: Looks like you might have done this and forgotten to remove the comment?

File: AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx [hunk 2]
```diff
@@ -30,10 +30,8 @@ export default class SearchResult extends React.Component {
 
     renderResultBody(result) {
         return Object.keys(result).map(key => {
-            if (result[key] == null || result[key] == '')
-            {return null;}
-            if (key == 'reportTitle')
-            {return null;}
+            if (result[key] == null || result[key] == ''){return null;}
+            if (key == 'reportTitle'){return null;}
```
Comment (RogVan) [inline AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx:34]: Is this lint formatting the code wrong? You might need to manually undo this formatting, and have it as:
```suggestion
            if (result[key] == null || result[key] == ''){
                return null;
            }
            if (key == 'reportTitle'){
                return null;
            }
```

**Review by GabrielParle — APPROVED** [review]
Delete route works nicely

**Review by PhillyJB — APPROVED** [review]
Seems all good to me. Delete Function Working fine for Locations, Regions and Users in Admin.

**Pull request merged**